src/index.js

```diff
@@ -4,6 +4,10 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import 'bootstrap/dist/css/bootstrap.css';
+
 const root = ReactDOM.createRoot(document.getElementById('root'));
 
 root.render(
```

Comment:
The only thing missing and the reason it's not working is that you forgot to wrap the App with Router
```
root.render(
  <React.StrictMode>
  <Router>
    <App />
  </Router>
  </React.StrictMode>
);
```